manual/src/refman/patterns.etex

```diff
@@ -88,6 +121,11 @@ parenthesized pattern, as in @"(" pattern_1 ":" typexpr ")"@. This
 constraint forces the type of @pattern_1@ to be compatible with
 @typexpr@.
 
+\begin{caml_example}{toplevel}
+let int_triple_is_ordered ((a, b, c) : int * int * int) =
+  a < b && b < c;;
```

Comment:
```
int_triple_is_ordered (1,1,2);;
- : bool = false
```
Question:  
Are two identical values not ordered  (and then a <=b && b <= c)?

Reply:
Fixed per your suggestion. Thanks.